includes/class-sensei-grading.php

```diff
@@ -52,24 +52,29 @@ public function __construct( $file ) {
 	}
 
 	/**
-	 * grading_admin_menu function.
+	 * Add the Grading submenu.
 	 *
 	 * @since  1.3.0
 	 * @access public
-	 * @return void
 	 */
 	public function grading_admin_menu() {
+		$indicator_html = '';
+		$grading_counts = Sensei()->grading->count_statuses( apply_filters( 'sensei_grading_count_statues', [ 'type' => 'lesson' ] ) );
```

Comment:
I found that we have a few usages of this hook, but we don't have a doc block for it anywhere. Maybe it is a good moment for it? ;)
Also, it looks like there is a mistype in the hook name :(

Reply:
Ah, good call. I've actually removed this filter in https://github.com/Automattic/sensei/pull/4671/commits/959845b48caa0aa394f04dc1ac47361ef74bb4e2 because I realized that it's being filtered _again_ in the [`count_statuses` function](https://github.com/Automattic/sensei/blob/master/includes/class-sensei-grading.php#L423). So if someone really wants to change the arguments, they can use that filter instead.